docker/gcp_pudl_etl.sh

```diff
@@ -95,17 +95,23 @@ ETL_SUCCESS=${PIPESTATUS[0]}
 
 # if pipeline is successful, distribute + publish datasette
 if [[ $ETL_SUCCESS == 0 ]]; then
-    # Dump outputs to s3 bucket if branch is dev or build was triggered by a tag
-    if [ $GITHUB_ACTION_TRIGGER = "push" ] || [ $GITHUB_REF = "dev" ]; then
-        copy_outputs_to_distribution_bucket
-        ETL_SUCCESS=${PIPESTATUS[0]}
-    fi
-
     # Deploy the updated data to datasette
     if [ $GITHUB_REF = "dev" ]; then
         python ~/devtools/datasette/publish.py 2>&1 | tee -a $LOGFILE
         ETL_SUCCESS=${PIPESTATUS[0]}
     fi
+
+    # Compress the SQLite DBs for easier distribution
+    gzip $PUDL_OUTPUT/*.sqlite
+
+    # Remove redundant multi-file EPA CEMS outputs prior to distribution
+    rm -rf $PUDL_OUTPUT/hourly_emissions_epacems/
```

Comment:
Outside the scope of this PR but one issue with having all of this workflow logic stored in script is that we might not get notified by some failures. For example, if the ETL is successful but zipping the sqlite or removing the epacems directory fails, we'll still get a "Successful" notification in slack.

A potential short term solution would be to update the `ETL_SUCCESS` variable:

```bash
gzip $PUDL_OUTPUT/*.sqlite && \
rm -rf $PUDL_OUTPUT/hourly_emissions_epacems/

ETL_SUCCESS=${PIPESTATUS[0]}
```

Reply:
Ahh okay yes I'll make this change.